package.json

```diff
@@ -14,14 +15,15 @@
     "react": "^0.13.3",
     "request": "2.64.0",
     "object-assign": "^4.0.1",
-    "react-dnd": "1.1.8"
+    "react-dnd": "1.1.8",
+    "browserify": "^13.0.0",
```

Comment:
Heroku only installs hard dependencies and not `dev-dependencies` so we have to have the build tools listed in this section.